lib/app/item.ex

```diff
@@ -168,6 +168,32 @@ defmodule App.Item do
     end)
   end
 
+  @doc """
+  `person_with_item_and_timer_count/0` returns a list of number of timers and items per person.
+  Used mainly for metric-tracking purposes.
+
+  ## Examples
+
+  iex> person_with_item_and_timer_count()
+  [
+    %{name: nil, num_items: 3, num_timers: 8, person_id: 0}
+    %{name: username, num_items: 1, num_timers: 3, person_id: 1}
+  ]
+  """
+  def person_with_item_and_timer_count() do
+    sql = """
+    select p.person_id, p.name, COUNT(distinct i.id) as "num_items", count(distinct t.id) as "num_timers"
+    from people p
```

Comment:
The `people` table should no longer exist in this project. https://github.com/dwyl/mvp/issues/118
Removing it _now_. ✂️